operator/src/main/java/org/keycloak/operator/controllers/KeycloakIngress.java

```diff
@@ -119,9 +121,9 @@ protected Ingress fetchExistingIngress() {
     }
 
     public void updateStatus(KeycloakStatusBuilder status) {
-        if (!keycloak.getSpec().isDisableDefaultIngress() && existingIngress == null) {
+        IngressSpec ingressSpec = keycloak.getSpec().getIngressSpec();
+        if (ingressSpec != null && ingressSpec.isIngressEnabled() && existingIngress == null) {
```

Comment:
As we assume the custom Nginx Ingress should be created by default, the 'notReady' message should also be included in the overall status when the spec/property is not defined and there's no existing Ingress, right?

The thing is that when the spec is not defined (and there's no existing Ingress) the message is not propagated to the status.